go/connection_test.go

```diff
@@ -24,14 +24,15 @@ import (
 	"context"
 	"database/sql"
 	"database/sql/driver"
+	credentials2 "github.com/aws/aws-sdk-go-v2/credentials"
```

Comment:
Can this just be `"github.com/aws/aws-sdk-go-v2/credentials"` and the 2 commented out imports be removed?

Reply:
Ah, yep, thanks